zeppelin-web/src/app/jobmanager/jobmanager.controller.js

```diff
@@ -98,10 +98,19 @@ function JobmanagerCtrl($scope, websocketMsgSrv, $interval, ngToast, $q, $timeou
       filterValueInterpreter: '*',
       isSortByAsc: true
     };
+    $scope.sortTooltipMsg = 'Switch to sort by desc';
     $scope.jobTypeFilter = jobManagerFilter;
 
     websocketMsgSrv.getNoteJobsList();
 
+    $scope.$watch('filterConfig.isSortByAsc', function (value) {
+      if (value) {
+        $scope.sortTooltipMsg = 'Switch to sort by desc';
+      } else {
+        $scope.sortTooltipMsg = 'Switch to sort by asc';
+      }
+    });
+
```

Comment:
Appreciate for taking my opinion @cloverhearts!

Maybe Ii's really minor suggestion, can line number 101, 107~111  be like below? 😀
```
$scope.sortTooltipMsg = value ? 'Switch to sort by desc' : 'Switch to sort by asc';
```